swagger.yaml

```diff
@@ -0,0 +1,367 @@
+swagger: "2.0"
+info:
+  description: "An API used to query information about datasets published by the ONS."
+  version: "1.0.0"
+  title: "ONS Dataset API"
+  license:
+    name: "Open Government Licence v3.0"
+    url: "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
+basePath: "/v1"
+tags:
+- name: "Dataset"
+schemes:
+- "http"
+parameters:
+  id:
+    name: id
+    description: "Id that represents a dataset type"
+    in: path
+    type: string
+    required: true
+  edition:
+    name: edition
+    description: "An edition of a dataset"
+    in: path
+    type: string
+    required: true
+  version:
+    name: version
+    description: "A version of a dataset dataset"
+    in: path
+    type: string
+    required: true
+  dimension:
+    name: dimension
+    description: "A dimension from a dataset"
+    in: path
+    type: string
+    required: true
+  filter:
+    name: filter
+    description: "Filter a dataset using its dimensions"
+    in: body
+    schema:
+      $ref: '#/definitions/Filter'
+    required: true
+  pageSize:
```

Comment:
We are reworking the pagation query. To use a offset and number of items per page